homeassistant/components/roku/config_flow.py

```diff
@@ -0,0 +1,148 @@
+"""Config flow for Roku."""
+import socket
+from typing import Any, Dict, Optional
+from urllib.parse import urlparse
+
+from roku import Roku, RokuException
+import voluptuous as vol
+
+from homeassistant.components.ssdp import (
+    ATTR_SSDP_LOCATION,
+    ATTR_UPNP_FRIENDLY_NAME,
+    ATTR_UPNP_SERIAL,
+)
+from homeassistant.config_entries import CONN_CLASS_LOCAL_POLL, ConfigFlow
+from homeassistant.const import CONF_HOST, CONF_NAME
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.typing import HomeAssistantType
+
+from .const import DOMAIN  # pylint: disable=unused-import
+
+DATA_SCHEMA = vol.Schema({vol.Required(CONF_HOST): str})
+
+ERROR_CANNOT_CONNECT = "cannot_connect"
+ERROR_UNKNOWN = "unknown"
+
+
+def get_ip(host: str) -> str:
+    """Translate hostname to IP address."""
```

Comment:
Why would we translate it to an IP address?

Reply:
ill remove it, i heavily based my PRs on the same code base so just need to sync this up. prob focus on this for 0.108.0 beta unless you would be ok doing it as a 0.107.0b1?

Reply:
let's push it out to 108. We want to avoid adding features to betas